src/main/java/Duke.java

```diff
@@ -1,10 +1,77 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    private static final String LS = System.lineSeparator();
+    private static final String S_TAB = "     ";
+
+    private static final String MESSAGE_WELCOME = S_TAB + "Welcome to Jura Tempest!" + LS
+                                                + S_TAB + "I'm Rimuru Tempest, pleased to make your acquaintance." + LS
+                                                + S_TAB + "How can I help you today?";
+    public static final String MESSAGE_GOODBYE = S_TAB + "Sayonara. Come visit our country again soon!";
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        greetUser();
+        executeResponse();
     }
```

Comment:
good refractoring here. Good jon at making your functions shorter and making your main function especially short.